.github/workflows/main.yml

```diff
@@ -3,6 +3,8 @@ name: CI-Build
 env:
   # Turn on msvc analysis during build, enable once warnings are clean.
   DETOURS_ANALYZE: true
+  # Enable feature for CI build
+  DETOURS_SUPPORT_CONTEXT_CALLBACK: true
```

Comment:
This should probably be a test matrix, to keep the non callback path tested.

Reply:
I was hesitant to double the number of workflows that were run (from 16 to 32).

I changed it to run the matrix, but exclude both ARM flavors as they are not supported (so 24 workflows).  The exclude can be removed easily if that is desired.